app/config.py

```diff
@@ -510,10 +511,16 @@ class Development(Config):
 
     NOTIFICATION_QUEUE_PREFIX = os.getenv("NOTIFICATION_QUEUE_PREFIX", "vanotify-")
 
+    # TODO raise an error in dev / prod if unable to connect instead of deafulting to localhost
     SQLALCHEMY_DATABASE_URI = os.getenv(  # nosec
         "SQLALCHEMY_DATABASE_URI",
         'postgresql://postgres@localhost/notification_api')
 
+    # TODO raise an error in dev / prod if unable to connect instead of deafulting to localhost
```

Comment:
Please create a ticket for this TODO and add the ticket number to this comment.

Reply:
I should've removed those comments because they're 1) dev setup 2) were left before i better understood how the system works